ComponentKit/Core/Scope/CKComponentScopeHandle.mm

```diff
@@ -190,6 +190,13 @@ - (void)resolve
 
 - (CKScopedResponder *)scopedResponder
 {
+  CKAssertFalse(_resolved);
```

Comment:
Looks pretty reasonable to me! I may move this assertion inside the if statement...